frontend/App.js

```diff
@@ -1,19 +1,10 @@
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import React from "react";
+import Viz from "./screens/DataScreen/viz";
```

Comment:
Should this be "./screens/VizScreen/viz

Either way we'll move this to ProductInfo later, but for now ;)

Reply:
Yes, please move your code from viz.js to HomeScreen/index.js and change that import to "./screens/HomeScreen". Then it should work and be in the Home screen, where we'll want to display it in the app :)

Reply:
Ok I will have it moved to the HomeScreen, and I will be looking to try to fix this web issue tonight.